.github/workflows/github-ci.yml

```diff
@@ -0,0 +1,36 @@
+name: GitHub Actions for Quantum Inspire Examples CI
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      max-parallel: 3
+      matrix:
+        python-version: [3.7, 3.8, 3.9]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install --upgrade .
+      - name: Run Tests
+        run: |
+          bash select_examples_to_execute.sh
+          jupyter --version
+          bash select_notebooks_to_execute.sh
```

Comment:
🔽 the bash scripts are quite small, if you think it would be better to lose them, I wouldn't mind having these as separate tasks and have the script in here. This of course if separate tasks doesn't add up too much throughput time.

Reply:
I think the bash scripts are oke to stay. They can be easily maintained seperately from the git actions.